.circleci/src/commands/@commands.yml

```diff
@@ -16,4 +16,12 @@ create_concatenated_patch_file:
   steps:
     - run:
         name: Combine patch-files to single file
-        command: ls -d -- packages/*/patches/*.patch | xargs cat > << parameters.filename >>
\ No newline at end of file
+        command: ls -d -- packages/*/patches/*.patch | xargs cat > << parameters.filename >>
+
+verify-common:
```

Comment:
Could you put this in it's own `@common-commands.yml` file? Would be better to keep `@commands.yml` as only root level commands

Reply:
yeah great idea